openshift/operator-controller/generate-manifests.sh

```diff
@@ -50,19 +50,20 @@ trap 'rm -rf $TMP_ROOT' EXIT
 
 # Copy all kustomize files into a temp dir
 cp -a "${REPO_ROOT}/config" "${TMP_ROOT}/config"
-mkdir -p "${TMP_ROOT}/openshift"
-cp -a "${REPO_ROOT}/openshift/kustomize" "${TMP_ROOT}/openshift/kustomize"
+mkdir -p "${TMP_ROOT}/openshift/operator-controller/"
+cp -a "${REPO_ROOT}/openshift/operator-controller/kustomize" "${TMP_ROOT}/openshift/operator-controller/kustomize"
 
 # Override OPENSHIFT-NAMESPACE to ${NAMESPACE}
-find "${TMP_ROOT}" -name "*.yaml" -exec sed -i "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
+find "${TMP_ROOT}" -name "*.yaml" -exec sed -i'.bak' "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
+
```

Comment:
As Todd mentioned in the other thread, I think we need to deal with the backup files that will be created after the `find ... sed` command runs.

Maybe add another command like this:
```
find "${TMP_ROOT}" -name "*.bak" -exec rm {} \;
```

Reply:
Let's see if it will pass the tests first, and then we will do that.

Reply:
Or add it to `.gitignore`

Reply:
@tmshort @joelanford 

I added a line to remove the .bak files, but since they’re created in TMP_ROOT, they’ll be cleaned up anyway when the trap deletes TMP_ROOT. So technically, the extra remove line isn’t really needed, but it’s there now, so it doesn’t hurt. 😊

Reply:
Yeah, if it's done in TMP_ROOT... so not necessary to clean up...